examples/dataset/objection_detection_2d/object_detection_2d/constants.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021-2024 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+BUCKET = "kolena-public-examples"
+DATASET = "coco-2014-val"
+ID_FIELDS = ["locator"]
+MODELS = [
+    "yolo_r",
+    "yolo_x",
+    "yolo_v3",
+    "yolo_v4s",
+    "faster_rcnn",
+    "mask_rcnn",
+]
+EVAL_CONFIG = {
+    "threshold_strategy": 0.5,
```

Comment:
Is there specific reason these values are different from those used in workflow/object_detection_2d example?

Reply:
Oh - these were arbitrarily chosen/selected. - will be updated (although they don't seem to really break the results)